### PR TITLE
feat(renderer): add a `setElementStyles` method

### PR DESCRIPTION
**Pull request by matsko**

### CODE REVIEW

**Comment by matsko** [comment]
LGTM from @mhevery

File: modules/angular2/test/web_workers/worker/renderer_integration_spec.ts [hunk 1]
```diff
@@ -162,6 +162,15 @@ export function main() {
                  renderer.setElementStyle(workerEl, 'width', null);
                  expect(DOM.getStyle(el, 'width')).toEqual('');
 
+                 renderer.setElementStyles(workerEl, {'height': '999px', 'opacity': '0.5'})
+                     expect(DOM.getStyle(el, 'height'))
```
Comment (gkalpak) [inline modules/angular2/test/web_workers/worker/renderer_integration_spec.ts]: Indentation is pretty weird in this block...

Comment (matsko) [inline modules/angular2/test/web_workers/worker/renderer_integration_spec.ts]: We use clang to format the code before its submitted so we don't have any control of the final output.

Comment (gkalpak) [inline modules/angular2/test/web_workers/worker/renderer_integration_spec.ts]: I know, but still :P Could it be that the missing `;` in the line above put clang-format off ?

Comment (mhevery) [inline modules/angular2/test/web_workers/worker/renderer_integration_spec.ts]: @matsko yes, it is the missing `;` please fix.

**Comment by mhevery** [comment]
@matsko what is the status of this PR? Should it be merged? Can you rebase and merge?

**Comment by matsko** [comment]
Landed as https://github.com/angular/angular/commit/1ac38bd69a9df4d7fbe6de1ee65657ebe89a9979

**Pull request closed without merging**